packages/cli/test/test-utils.js

```diff
@@ -63,10 +63,13 @@ exports.executeGenerator = function(GeneratorOrNamespace, settings) {
  * @property {boolean} excludeDataSourcesDir Excludes the datasources directory
  * @property {boolean} includeDummyModel Creates a dummy model file in /src/models/product-review.model.ts
  * @property {boolean} includeDummyRepository Creates a dummy repository file in /src/repositories/bar.repository.ts
+ * @property {boolean} includeSandboxFilesFixtures creates files specified in SANDBOX_FILES array
+ * @param {Object[]} sandBoxFiles specify files, directories and their content to be included as fixtures
  */
-exports.givenLBProject = function(rootDir, options) {
+exports.givenLBProject = function(rootDir, options, sandBoxFiles) {
```

Comment:
I find it redundant that we have to specify both `options.includeSandboxFilesFixture` and `sandBoxFiles`. If `sandBoxFiles` is not provided or empty, then `options.includeSandboxFilesFixture` can be considered as `false`. Similarly, if `sandBoxFiles` are provided and not empty, then `options.includeSandboxFilesFixture` can be considered as true. Any other combination does not make sense to me and I would consider it as a programmer's error.

I am proposing to merge `options.includeSandboxFilesFixture` and `sandBoxFiles` into `options.additionalFiles`.

Example usage:

```ts
testUtils.givenLBProject(
  SANDBOX_PATH,
  {additionalFiles: SANDBOX_FILES}
);
```